prow/plugins/lgtm/lgtm_test.go

```diff
@@ -592,51 +606,26 @@ func TestLGTMFromApproveReview(t *testing.T) {
 	}
 }
 
-type fakeIssueComment struct {
```

Comment:
Why are we losing this?

Reply:
I had to move to a proper `fakeGitHub` here because I have changed the first argument type of `handlePullRequest()` to a real `githubClient`.